blocks/subscriptions-block/components/ContactInfo/index.jsx

```diff
@@ -0,0 +1,158 @@
+	// const getTranslatedCountries = countryCodes.map((entry) => ({
+	// 	code: entry.code,
+	// 	name: phrases.t(entry.key),
+	// }));
+
+	const signOutBtn = (
+		<Button variant="secondary" size="medium" onClick={handleLogout}>
+			{phrases.t("checkout-block.identity-sign-out")}
+		</Button>
+	);
+
+	const identityInfo = () => {
+		if (user && signedInIdentity) {
+			const type = signedInIdentity.type.toLowerCase();
+			switch (type) {
+				case "google":
+					return (
+						<div className={`${className}__identity-row`}>
+							{/* {TO-DO: Change the Icon to Google <Icon name="Google"/> } */}
+							<Image alt="Google" title="Google" src={GoogleIcon} />
+							<span>
+								{phrases.t("checkout-block.identity-social", {
+									email: user.email,
+								})}
+							</span>
+							{signOutBtn}
+						</div>
+					);
+				case "facebook":
+					return (
+						<div className={`${className}__identity-row`}>
+							<Icon name="Facebook"/>
+							<span>
+								{phrases.t("checkout-block.identity-social", {
+									email: user.email,
+								})}
+							</span>
+							{signOutBtn}
+						</div>
+					);
+				case "password":
+					return (
+						<div className={`${className}__identity-row`}>
+							<span>
+								{phrases.t("checkout-block.identity-email", {
+									email: user.email,
+								})}
+							</span>
+							{signOutBtn}
+						</div>
+					);
+				default:
+					return null;
+			}
+		}
+		return null;
+	};
+
+	return (
+		<form onSubmit={handleSubmit} ref={formRef} className={`${className}__contact-info`}>
+			<Heading>{phrases.t("checkout-block.contact-info")}</Heading>
+			<Stack>
+				<Input
+					autoComplete="email"
+					label={phrases.t("checkout-block.email")}
+					name="email"
+					defaultValue={user && user.email ? user.email : ""}
+					hidden={!!(user && user.email)}
+					required
+					onChange={(value) => {
+						handleInputChange("email", value);
+					}}
+					showDefaultError={false}
+					type="email"
+					validationErrorMessage={phrases.t("checkout-block.email-requirements")}
+				/>
+				{identityInfo()}
+			</Stack>
+			<div className={`${className}__contact-info-profile`}>
+				<Input
+					label={phrases.t("checkout-block.first-name")}
+					name="firstName"
+					defaultValue={user && user.firstName ? user.firstName : ""}
+					required
+					onChange={(value) => {
+						handleInputChange("firstName", value);
+					}}
+					showDefaultError={false}
+					type="text"
+					validationErrorMessage={phrases.t("checkout-block.first-name-requirements")}
+				/>
+				<Input
+					label={phrases.t("checkout-block.last-name")}
+					name="lastName"
+					defaultValue={user && user.lastName ? user.lastName : ""}
+					required
+					onChange={(value) => {
+						handleInputChange("lastName", value);
+					}}
+					showDefaultError={false}
+					type="text"
+					validationErrorMessage={phrases.t("checkout-block.last-name-requirements")}
+				/>
+			</div>
+			<Input
```

Comment:
@LauraPinilla , I just merged in the dropdown changes to `arc-themes-components-version-2.1.2`, so if you pull down that branch locally you can update this input. You just need to change the `type` to `select` and add an array of options objects like so:
```
options={[
  { label: 'Option 1', value: 'option1' },
  { label: 'Option 2', value: 'option2' },
]}
```

Reply:
You can also set the `optionLabelKey` and `optionValueKey` props to `key` and `code` respectively, to use the country codes in `countryCodes.js`. Just pass the `key` value through `phrases.t()` first.

Reply:
You can also set the `optionLabelKey` and `optionValueKey` props to `key` and `code` respectively, to use the country codes in `countryCodes.js`. Just pass the `key` value through `phrases.t()` first.